pkg/nad/masterplugin.go

```diff
@@ -393,3 +393,161 @@ func (plugin *MasterIPVlanPlugin) GetMasterPluginConfig() (*MasterPlugin, error)
 
 	return plugin.masterPlugin, nil
 }
+
+// MasterBondPlugin provides struct for MasterPlugin to create a NAD for bonded interfaces.
+type MasterBondPlugin struct {
+	masterPlugin *MasterPlugin
+	errorMsg     string
+}
+
+// NewMasterBondPlugin define network-attachment-interface for bond interface.
```

Comment:
wrong  comment. Should be something like:
// NewMasterBondPlugin creates new instance of MasterBondPlugin

Reply:
changed